examples/tomcat/src/main/scala/com/example/http4s/tomcat/TomcatSslExample.scala

```diff
@@ -1,9 +1,17 @@
 package com.example.http4s
 package tomcat
 
-import com.example.http4s.ssl.SslExample
-import org.http4s.server.tomcat.TomcatBuilder
+import org.http4s.server.ServerApp
+import org.http4s.server.tomcat.TomcatConfig
+import org.http4s.tls.TlsConfig
 
-object TomcatSslExample extends SslExample {
-  def builder = TomcatBuilder
+object TomcatSslExample extends ServerApp {
+  val tlsConfig = TlsConfig.default
+    .withKeyStore(".././keystore")
```

Comment:
Not the double dots, the single dot.  I expected `../keystore`, not `.././keystore`.